Source/WebKit/UIProcess/WebProcessProxy.messages.in

```diff
@@ -102,8 +102,8 @@ messages -> WebProcessProxy WantsDispatchMessage {
 #endif
 
 #if ENABLE(REMOTE_INSPECTOR) && PLATFORM(COCOA)
-    CreateServiceWorkerDebuggable(WebCore::ServiceWorkerIdentifier identifier, URL url)
+    CreateServiceWorkerDebuggable(WebCore::ServiceWorkerIdentifier identifier, URL url, enum:bool WebCore::ServiceWorkerIsInspectable isInspectable) -> (bool shouldWaitForAutoInspection) Synchronous
```

Comment:
Do we actually expect/need this to be synchronous if the debuggable thread is going to spin and wait anyway?

Reply:
This actually works either being synchronous or async. From the rest of [`WebProcessProxy.messages.in`](https://github.com/WebKit/WebKit/blob/c160b11edda887fab8467eb9e37cf06e68d95fb2/Source/WebKit/UIProcess/WebProcessProxy.messages.in) it looked like we have preferred synchronous replies, and it avoided the need for another completion callback.

Did you think async would make more sense though?

Reply:
We should always avoid sync IPC whenever possible.